src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep firstArg:'one', secondArg:'two'"));
```

Comment:
:bug: hold on, this does not make sense to me.

In `SnippetizerTest` you are asserting that `Snippetizer` will produce

```groovy
monomorphStep([firstArg: 'one', secondArg: 'two'])
```

which is a syntax that has always worked in `DSL` (AFAIK), so we are just making sure that users are offered that instead of

```groovy
monomorphStep [firstArg: 'one', secondArg: 'two']
```

which did not work—fine.

But now here you seem to be patching `DSL` so that it will also accept a new and simplified syntax

```groovy
monomorphStep firstArg: 'one', secondArg: 'two'
```

That may be fine—I do not completely follow the details of that part of the patch, and probably you should read and delete [this comment from @kohsuke](https://github.com/jenkinsci/workflow-cps-plugin/blob/f5e4f4eafee42f1096a33baf18bfe69726d6d1e5/src/main/java/org/jenkinsci/plugins/workflow/cps/Snippetizer.java#L296-L324) if you want to go that way—but then surely `Snippetizer` should also be patched to _generate_ the simplified syntax?